templates/html.html.twig

```diff
@@ -47,6 +47,13 @@
     <title>{{ head_title|safe_join(' | ') }}</title>
     <css-placeholder token="{{ placeholder_token }}">
     <js-placeholder token="{{ placeholder_token }}">
+    <script>
```

Comment:
This should go into a script file and added through a library. I can show you this if you want to zoom session this change.

Reply:
Oops, I can actually take this out.  I am not sure that I really need to have it on the live site at all? I think Google Tag Manager might be handling it. Hm, although maybe better safe than sorry...

Reply:
Will take you up on your offer. Assuming it goes in js/vendor ?